packages/data-store/src/merge-data.js

```diff
@@ -4,23 +4,51 @@ export default function mergeData(currentData, newData, action) {
 
   let result;
 
+  // if (action == 'create' && newData.id) action = 'update'; // return call na de POST
```

Comment:
Is deze comment nog relevant?

Reply:
Ja, die staat als issue in Trello. Ga ik later fixen.